src/client/connect.rs

```diff
@@ -13,7 +13,7 @@ use std::error::Error;
 use std::fmt;
 use std::marker::PhantomData;
 
-/// Establishes an H2 client connection.
+/// Establishes an H2 client connection.\
```

Comment:
Why this change?

Reply:
That looks like a typo on my part, thanks for catching it!